cache/edge_lambdas/src/redirects.ts

```diff
@@ -102,8 +101,7 @@ export const literalRedirects: Record<string, string> = {
   '/whats-on/exhibitions/brains': '/exhibitions/W31EzykAACkAPuXG',
   '/whats-on/exhibitions/infinitas-gracias': '/exhibitions/W31D3CkAACIAPuGN',
   '/whats-on/exhibitions/superhuman': '/exhibitions/W31FpSkAACkAPumK',
-  '/young-people': '/pages/Wuw2MSIAACtd3Ssg',
```

Comment:
is this removing `/young-people` altogether?

Reply:
Oh yeah you're right, it should be `/young-people`: `pages/young-people`, thank you!

Reply:
Addressed!